exercises/kindergarten-garden/description.md

```diff
@@ -1,19 +1,19 @@
 Given a diagram, determine which plants each child in the kindergarten class is
 responsible for.
 
-The kindergarten class is learning about growing plants. The teachers
+The kindergarten class is learning about growing plants. The teacher
 thought it would be a good idea to give them actual seeds, plant them in
 actual dirt, and grow actual plants.
 
 They've chosen to grow grass, clover, radishes, and violets.
 
-To this end, they've put little Styrofoam cups along the window sills,
-and planted one type of plant in each cup, choosing randomly from the
-available types of seeds.
+To this end, the children have put little cups along the window sills, and 
+planted one type of plant in each cup, choosing randomly from the available 
+types of seeds.
 
 ```text
 [window][window][window]
-........................ # each dot represents a Styrofoam cup
+........................ # each dot represents a cup location
```

Comment:
"location" is redundant here.

The usage also grates a little with me, since location is about where something is.

Q: What does this dot represent? `.`

a) A cup
b) A cup location.

if b, where is the cup located?

Reply:
I agree with you here.  I was on the fence about adding `location`.  Kinda felt right, kinda didn't, so I thought I would try it on for size.  :)  I will remove it.